### ISSUE TITLE
Inconsistency of Axis Labels in Confusion Matrices for Chapter 5 on Scikit-Learn

### ISSUE THREAD

**Issue by stefmolin** [jakevdp/PythonDataScienceHandbook#130]
The first time the confusion matrix is created, the x-axis is labeled as "predicted value" and the y-axis as "true value"; however, all other confusion matrices are the reverse.  I am genuinely confused as to which is correct now, and this makes a big deal when interpreting the results. From what I have seen in the sklearn documentation, the first definition (predicted on the x-axis) is correct, but I wanted some confirmation that the rest were errors.

**Comment by stefmolin** [comment]
Pull request here: https://github.com/jakevdp/PythonDataScienceHandbook/pull/135

**Comment by djvera84** [comment]
Hello @stefmolin and @jakevdp,

I was confused about this myself but I actually think it is correct as written.  The documentation says that the confusion matrix :math:`C` is such that :math:`C_{i, j}` is equal to the number of observations known to be in group :math:`i` but predicted to be in group :math:`j`, so the vertical labels are actually labeling the 'rows' i and the horizontal labels are labeling the columns. For example, in the matrix in question, row 2 consists of data with the ground truth classification of 2; column 8 (predicted) 8 shows digits predicted to be an 8. So :math:`C_{2, 8}` = 15 shows that 15 2's were classified as 8. The heatmap for the confusion matrix on page 388, 419, etc, all transpose the confusion matrix beforehand so the labels would be reversed, if I am understanding correctly. 

Also, thank you @jakevdp for your excellent book.
